resources/definitions/syndaveraxi.def.json

```diff
@@ -0,0 +1,46 @@
+{
+    "version": 2, 
+    "name": "SyndaverAXI",
+    "inherits": "fdmprinter",
+    "metadata": 
+{
+        "type": "machine",
+        "visible": true,
+        "author": "Syndaver3D",
+        "manufacturer": "Syndaver3D",
+        "file_formats": "text/x-gcode",
+        "supports_usb_connection": true,
+        "preferred_quality_type": "draft",
+        "machine_extruder_trains":
+        {
+            "0": "syndaveraxi_extruder_0"
+        }
+    },
+
+    "overrides": {
+        "machine_name": { "default_value": "AXI" },
+        "machine_shape": { "default_value": "rectangular"},
+        "machine_heated_bed": { "default_value": true },
+        "machine_width": { "default_value": 280 },
+        "machine_depth": { "default_value": 280 },
+        "machine_height": { "default_value": 286 },
```

Comment:
A bit of a nitpicking, but the official product dimensions are 280x280x285. Is the extra milimeter in the height intended?